tests/lib/rules/jsx-pascal-case.js

```diff
@@ -70,5 +70,17 @@ ruleTester.run('jsx-pascal-case', rule, {
   }, {
     code: '<YMCA />',
     errors: [{message: 'Imported JSX component YMCA must be in PascalCase'}]
+  }, {
+    code: '<_TEST_COMPONENT />',
+    options: [{allowAllCaps: true}],
+    errors: [{message: 'Imported JSX component _TEST_COMPONENT must be in PascalCase'}]
+  }, {
+    code: '<TEST_COMPONENT_ />',
+    options: [{allowAllCaps: true}],
+    errors: [{message: 'Imported JSX component TEST_COMPONENT_ must be in PascalCase'}]
```

Comment:
seems like the message, when `allowAllCaps`, should say "in PascalCase or SCREAMING_SNAKE_CASE"?